lib/zendesk_apps_support/manifest.rb

```diff
@@ -50,30 +42,31 @@ def no_template_locations
     end
 
     def location?
-      locations != { 'zendesk' => {} }
+      !locations.values.all?(&:empty?)
     end
 
     def locations
       @locations ||=
         case original_locations
         when Hash
-          original_locations
+          replace_legacy_locations! original_locations
         when Array
-          { 'zendesk' => NoOverrideHash[original_locations.map { |location| [ location, LEGACY_URI_STUB ] }] }
+          { 'support' => NoOverrideHash[original_locations.map { |location| [ location, LEGACY_URI_STUB ] }] }
         when String
-          { 'zendesk' => { original_locations => LEGACY_URI_STUB } }
+          { 'support' => { original_locations => LEGACY_URI_STUB } }
         # TODO: error out for numbers and Booleans
         else # NilClass
-          { 'zendesk' => {} }
+          { 'support' => {} }
         end
+    rescue OverrideError => error
+      # if the error contains the word `_legacy` in the second sentence, let's
+      # only use the first one.
+      error.message = error.message[/([^.]*)\./] if error.message[/_legacy/]
```

Comment:
This code knows a lot about the error message format. Could we expose a couple more methods on the OverrideError class so we could write something like:

```
if error.values.any {|val| val =~ /_legacy/ }
  error.suppress_values!
end
```

(internally error could track `key_message` and `values_message`, then the `message` getter could just include the first when suppress_values has been set).